deep_qa/training/step.py

```diff
@@ -0,0 +1,81 @@
+
+from typing import List
+
+import tensorflow
+import numpy
+import keras.backend as K
+
+
+class Step:
+    """Runs a computation graph.
+
+    # Arguments
```

Comment:
Formatting here should be numpydoc.

Or we need to just switch to markdown...  But for now, everything's rST / numpydoc, so we should be consistent.